active_learning/learning.py

```diff
@@ -42,12 +79,32 @@ def classify_svm(self, X, y):
         print("Accuracy: %0.2f (+/- %0.2f)" % (scores.mean(), scores.std() * 2))
         return scores
 
+
     def learn(self, auto_label=True, active_learning_strategy=5, num_queries=50, train_size=0.005):
+        '''
+        Uses the specified learning strategy and then queries the oracle/learner before printing the accuracy at which the articles are labeled
+
+        Parameters
+        ----------
+        self : self reference
```

Comment:
remove this, as we don't actually pass a self reference when calling this function